Build test warnings should be fixed

currently, some warnings are raised in the tests:
```
[92/92] Linking target test/search.
ninja: Entering directory `build'
ninja: warning: bad deps log signature or version; starting over
[21/93] Compiling C++ object 'test/9f86d08@@find@exe/find.cpp.o'.
../test/find.cpp: In member function 'virtual void {anonymous}::FindTests_ByPath_Test::TestBody()':
../test/find.cpp:160:15: warning: unused variable 'entry' [-Wunused-variable]
  160 |     for(auto& entry: range5) {
      |               ^~~~~
../test/find.cpp:167:15: warning: unused variable 'entry' [-Wunused-variable]
  167 |     for(auto& entry: range6) {
      |               ^~~~~
[38/93] Generating resources with a custom command.
```
Can be fixed easily by finding the number of articles differently.

--- a/test/find.cpp
+++ b/test/find.cpp
@@ -155,20 +155,19 @@ TEST(FindTests, ByPath)
     }
     ASSERT_EQ(count, 34);
 
-    ASSERT_EQ(range5.begin()->getIndex(), 0);
     count = 0;
     for(auto& entry: range5) {
+      ASSERT_EQ(count, entry.getIndex());
       count++;
     }
     ASSERT_EQ(count, 118);
 
-    ASSERT_EQ(range6.begin()->getIndex(), 0);
     count = 0;
     for(auto& entry: range6) {
+      ASSERT_EQ(count, entry.getIndex());
       count++;
     }
     ASSERT_EQ(count, 118);
-    
 }
 
 } // namespace

